### ISSUE TITLE
Could you please change the Typio license from Creative Commons to MIT, so that we can make and publish changes to Typio?

### ISSUE THREAD

**Issue by unforgettableid** [ctsstc/typio-form-recovery-reboot#6]
Dear @nicklassandell:

## Introduction

Thank you for writing Typio Form Recovery!  It doesn't work anymore, due to changes to Google Chrome.  @ctsstc hopes to fix it and get it working again.

## The problem

Unfortunately, you licensed Typio as "Creative Commons Attribution-NonCommercial-NoDerivatives 4.0".  This means that we are legally forbidden from compiling and distributing a fixed version to the public.

## Proposed solution

Would you be willing to please relicense Typio under a more-liberal license, such as the MIT license?

Legally, the best way to do so might be to edit https://bitbucket.org/nicklassandell/chrome-form-recovery/src/master/license.txt so that it holds the new license.

## Conclusion

I'd be curious to hear your thoughts.

Thank you for reading this!

**Comment by ctsstc** [comment]
Yeah it seems that there's no response from Nick from what I've seen. I knew I needed to dig through and understand the license better, but it does seem that maybe the best thing to do is start something from scratch in memory of Typio unfortunately. The code is quite dated, I was still running into issues trying to get it all running and rendering properly, although there was a large effort put in on updating it and getting it to run in the first place. I'm not very experienced with Vue and updating it from v2 to v3 as well as the manifest from v2 to v3 was a bit of a lift. Typio is a great extension, but it seems that the best route may be to start something fresh unfortunately. At this point I'm not sure if the owner is still around. I had ambitions of carrying the torch, but it didn't feel right to swap out or close some attributions nor would it be valid in the current license. For all we know, people are donating to a black hole currently which isn't great either.

**Comment by dumblob** [comment]
We can at least cement the Nick's code under CC and everything new under MIT (both are compatible AFAIK). And later once (if at all) Nick appears around, we can ask him to relicense also his work under MIT to not have this repo dual-licensed.

**Comment by unforgettableid** [comment]
@dumblob:

I found [this discussion](https://opensource.stackexchange.com/questions/357/is-the-cc-by-nd-an-open-license) on Creative Commons NoDerivatives clauses.  They're very restrictive, and they seem to me at least to strongly discourage forking.

I think the best thing to do would be for us to keep trying to contact Nicklas, with maybe one ping every 4-12 months at first.  There are [multiple](https://codepen.io/nicklassandell) [ways](https://twitter.com/nicklassandell/with_replies) to [contact](https://github.com/nicklassandell) him.  I think it'd best to try a variety of different contact methods.

A good email subject line might be:  "Typio relicensing request: CC-BY-NC-ND > MIT. Fine with you?"

**Comment by unforgettableid** [comment]
@dumblob:

A possible alternative to Typio which we could use for now is [Formalizr](https://chrome.google.com/webstore/detail/formalizr/ceoiohgolheciaefomofjboicekphfmp).  It uses the Apache License 2.0, according to its [license declaration](https://github.com/dennisbabkin/Formalizr/blob/master/LICENSE).

**Comment by dumblob** [comment]
Hm, interesting, the license was actually added by @nisansei in https://github.com/ctsstc/typio-form-recovery-reboot/commit/39cf142754428ed3076c832caa8812575737b485 .

**Comment by ctsstc** [comment]
I think the original two comitters are the same person.

**Comment by dfinchau** [comment]
> @dumblob:
> 
> A possible alternative to Typio which we could use for now is [Formalizr](https://chrome.google.com/webstore/detail/formalizr/ceoiohgolheciaefomofjboicekphfmp). It uses the Apache License 2.0, according to its [license declaration](https://github.com/dennisbabkin/Formalizr/blob/master/LICENSE).

Sadly Formalizr has not been updated for ~5 years and is not compatible with manifest v3...
